modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbTableStorage.java

```diff
@@ -87,53 +73,42 @@
     /** Data region for the table. */
     private final RocksDbDataRegion dataRegion;
 
-    /** Comparators factory for indexes. */
-    private final BiFunction<TableView, String, Comparator<ByteBuffer>> indexComparatorFactory;
-
     /** List of closeable resources to close on {@link #stop()}. Better than having a field for each one of them. */
     private final List<AutoCloseable> autoCloseables = new ArrayList<>();
 
-    /** Rocks DB instance itself. */
+    /** Rocks DB instance. */
     private RocksDB db;
 
     /** CF handle for meta information. */
     @SuppressWarnings("unused")
-    private ColumnFamilyHandle metaCfHandle;
+    private ColumnFamily metaCf;
```

Comment:
Why is it unused tho?

Reply:
This field was introduced by @ibessonov and he states that it is reserved for future purposes... Or do you mean that we should use it somehow?